uamqp/client.py

```diff
@@ -75,6 +77,8 @@ class AMQPClient(object):
     :param idle_timeout: Timeout in milliseconds after which the Connection will close
      if there is no further activity.
     :type idle_timeout: int
+    :param auth_timeout: Timeout in seconds for authentication
```

Comment:
Maybe clarify that this is the timeout for token negotiation in the case of CBS auth? It will otherwise be ignored, right?

Reply:
We should probably also document what the default value that we're using is.

Reply:
your understanding is correct, doc updated